.github/workflows/linux-package.yml

```diff
@@ -0,0 +1,77 @@
+name: Linuax Package
+on:
+  push:
+    tags:
+      - 'opentrack-\d+\.\d+\.\d+'
+  workflow_dispatch:
+env:
+  build_type: Release
+  onnx_version: 1.21.0
+  onnx_prefix: onnxruntime-linux-x64
+  file_name: '' # to disable warning in case of a linter
+  Qt5_DIR: '' # to disable warning in case of a linter
+jobs:
+  Package:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Clone opentrack/opentrack
+      uses: actions/checkout@v4
+    - name: Clone opentrack/depends
+      uses: actions/checkout@v4
+      with:
+        repository: opentrack/opentrack-depends
+        submodules: true
+        path: opentrack-depends
+
+    - name: Install Dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install libprocps-dev libopencv-dev wine64-tools
+        wget -q https://github.com/microsoft/onnxruntime/releases/download/v${{env.onnx_version}}/${{env.onnx_prefix}}-${{env.onnx_version}}.tgz
+        tar xzf ${{env.onnx_prefix}}-${{env.onnx_version}}.tgz
+    - name: Install Qt
+      uses: jurplel/install-qt-action@v4
+      with:
+        version: 5.15
+        archives: icu qtbase qttools qtserialport
+
+    - name: Configure and Build
+      run: |
+        cmake -S . -B build \
+          -DCMAKE_BUILD_TYPE=${{env.build_type}} \
+          -DONNXRuntime_DIR=${{env.onnx_prefix}}-${{env.onnx_version}} \
+          -DQt5_DIR=${{env.Qt5_DIR}}
```

Comment:
The official documentation actually recommends using `CMAKE_PREFIX_PATH`, over `Qt5_DIR`, see [here](https://doc.qt.io/qt-5/cmake-get-started.html).

Reply:
It's probably good enough. But at least do this for correctness' sake.

```cmake
list(APPEND CMAKE_PREFIX_PATH ${{env.Qt5_DIR}})
```

Reply:
It's not a CMake file though.